spec/signup_spec.rb

```diff
@@ -132,6 +132,17 @@
 		expect(page.status_code).to eq(400)
 	end
 
+  it "should require a @location.police.uk or @police.uk email for person_email" do
```

Comment:
`location.police.uk` is now covered by the more general `police.uk` domain, I think?  No need to specify in that case